docs/userguide/configuration.rst

```diff
@@ -2311,7 +2314,7 @@ certificate authority:
 _________
 
 
-The setting must be a dict the keys:
+The setting must be a dict with the following keys:
 
 *  ``ssl_cert_reqs`` (required): one of the ``SSLContext.verify_mode`` values:
     * ``ssl.CERT_NONE``
```

Comment:
In the current version a string should be used, so `"CERT_NONE"`. After the resolution of https://github.com/celery/celery/issues/5371#issuecomment-472012315 is merged, probably both value types will be supported.

Reply:
Let's fix that so we can merge this.